anndata/_core/anndata.py

```diff
@@ -1422,13 +1422,10 @@ def copy(self, filename: Optional[PathLike] = None) -> 'AnnData':
                 varm=self.varm.copy(),
                 obsp=self.obsp.copy(),
                 varp=self.varp.copy(),
+                raw=self.raw,  # copies automatically
```

Comment:
I don't think the constructor should copy automatically. That leads to redundant memory usage which isn't always needed. This was part of the rework of IO and why reading is faster and uses much less memory now.

Reply:
It only copies when passed another AnnData’s Raw. So what you say is that it should only do a flat copy?

Reply:
Yes